src/app/public/src/modules/nav/nav.component.html

```diff
@@ -3,7 +3,7 @@
     <li class="stache-nav-list-item"
       *ngFor="let route of routes"
       [attr.aria-current]="route.isCurrent ? 'page' : false"
-      [ngClass]="{'active': route.isActive, 'current': route.isCurrent}">
+      [ngClass]="{'active': route.isActive, 'current': route.isCurrent, 'stache-table-of-contents-active': route.isActiveTocAnchor}">
```

Comment:
Can we just make use of one of these already existing classes instead of adding a new one?  `isCurrent` would make sense in this situation?